PPI/manual_work/Jul_2023/manual_tables.sql

```diff
@@ -0,0 +1,383 @@
+    END
+$body$;
+
+SELECT * FROM dev_ppi.concept_relationship_manual_backup_2023_07_25;
+
+TRUNCATE concept_manual;
+TRUNCATE concept_relationship_manual;
+
+-- deprecate previous Mental Health and Well-Being Module concepts --229
+INSERT INTO dev_ppi.concept_manual (
+SELECT concept_name,
+       domain_id,
+       vocabulary_id,
+       concept_class_id,
+       null as standard_concept,
+       concept_code,
+       valid_start_date,
+       current_date as valid_end_date,
+       'D' as invalid_reason
+FROM concept where concept_code = 'mhwb'
+
+UNION
+
+SELECT c2.concept_name,
+       c2.domain_id,
+       c2.vocabulary_id,
+       c2.concept_class_id,
+       null as standard_concept,
+       c2.concept_code,
+       c2.valid_start_date,
+       current_date as valid_end_date,
+       'D' as invalid_reason
+FROM dev_ppi.concept c
+LEFT JOIN concept_relationship cr on c.concept_id = cr.concept_id_1
+LEFT JOIN concept c2 on cr.concept_id_2 = c2.concept_id
+where
+     c.concept_code = 'mhwb'
+and cr.relationship_id = 'PPI parent code of'
+
+UNION
+
+SELECT
+   c3.concept_name,
+   c3.domain_id,
+   c3.vocabulary_id,
+   c3.concept_class_id,
+   null as standard_concept,
+   c3.concept_code,
+   c3.valid_start_date,
+   current_date as valid_end_date,
+   'D' as invalid_reason
+FROM dev_ppi.concept c
+LEFT JOIN concept_relationship cr on c.concept_id = cr.concept_id_1
+LEFT JOIN concept_relationship cr2 on cr.concept_id_2 = cr2.concept_id_1
+LEFT JOIN concept c3 on cr2.concept_id_2 = c3.concept_id
+where
+c.concept_code = 'mhwb'
+and cr.relationship_id = 'PPI parent code of'
+and cr2.relationship_id = 'PPI parent code of'
+
+UNION
+
+SELECT
+   c4.concept_name,
+   c4.domain_id,
+   c4.vocabulary_id,
+   c4.concept_class_id,
+   null as standard_concept,
+   c4.concept_code,
+   c4.valid_start_date,
+   current_date as valid_end_date,
+   'D' as invalid_reason
+FROM dev_ppi.concept c
+LEFT JOIN concept_relationship cr on c.concept_id = cr.concept_id_1
+LEFT JOIN concept_relationship cr2 on cr.concept_id_2 = cr2.concept_id_1
+LEFT JOIN concept_relationship cr3 on cr2.concept_id_2 = cr3.concept_id_1
+LEFT JOIN concept c4 on cr3.concept_id_2 = c4.concept_id
+where
+c.concept_code = 'mhwb'
+and cr.relationship_id = 'PPI parent code of'
+and cr2.relationship_id = 'PPI parent code of'
+and cr3.relationship_id in ('Has answer (PPI)', 'Has question source'));
+
+-- deprecate previous Mental Health and Well-Being Module relationships
+INSERT INTO dev_ppi.concept_relationship_manual (
+SELECT c.concept_code,
+       c1.concept_code,
+       c.vocabulary_id,
+       c1.vocabulary_id,
+       cr.relationship_id,
+       cr.valid_start_date,
+       current_date as valid_end_date,
+       'D' as invalid_reason
+FROM concept c
+LEFT JOIN concept_relationship cr on c.concept_id = cr.concept_id_1
+LEFT JOIN concept c1 on cr.concept_id_2 = c1.concept_id
+WHERE c.concept_code in (SELECT concept_code FROM concept_manual)
```

Comment:
Never ever use concept code alone for the identification of concepts. Currently, all the manual tables are recreated by fastrecreate, therefore, this code is very dangerous